src/main.rs

```diff
@@ -42,7 +48,8 @@ fn main() {
 
     debug!(?args);
 
-    let database = Database::new_from_online();
+    let ureq_agent = UreqAgent::new(args.home_proxy, args.home_proxy);
```

Comment:
i need this pullrequest, but when i test it, i found this should be 
` let ureq_agent = UreqAgent::new(args.home_proxy, args.system_proxy);`

Reply:
Thanks for your review! I'll fix it tonight.